### PR TITLE
merge and gc as task

### PR DESCRIPTION
**Pull request by hermeGarcia**
### Description
Describe the proposed changes made in this PR.

### How was this PR tested?
Describe how you tested this PR.

### CODE REVIEW

**Comment by tarekziade** [comment]
Is there a way to trigger GCs during one of our benchmarks to check how it behaves on high load?

File: nucliadb_node/src/shards/scheduler.rs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+use nucliadb_core::NodeResult;
+use tokio::sync::{Semaphore, SemaphorePermit};
+use tokio::time::{self, Duration, Interval};
+
+use crate::shards::shard_writer::ShardWriter;
+
+// Every three hours a merge is triggered
+const MERGE_INTERVAL_SECS: u64 = 10800;
+// Everyday a shard is garbage collected
+const GC_INTERVAL_SECS: u64 = 10800;
```
Comment (tarekziade) [inline nucliadb_node/src/shards/scheduler.rs:32]: that's also 3 hours in seconds, and your comment says every day. Tipo?

Comment (hermeGarcia) [reply nucliadb_node/src/shards/scheduler.rs]: Yes tipo.. sorry about that!

**Comment by hermeGarcia** [comment]
> Is there a way to trigger GCs during one of our benchmarks to check how it behaves on high load?

Maybe we could do this through the grpc function?

**Review by vangheem — APPROVED** [review]

**Review by vangheem — APPROVED** [review]

**Comment by tarekziade** [comment]
supersed by https://github.com/nuclia/nucliadb/pull/new/merge-gc-task-2

**Pull request merged**